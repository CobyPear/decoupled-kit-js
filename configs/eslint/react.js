/** @type {import('eslint').ESLint.ConfigData} */
module.exports = {
	plugins: ['react'],
	extends: [
		'./index.js',
		'plugin:react/recommended',
		'plugin:react-hooks/recommended',
		'plugin:react/jsx-runtime',
	],
	settings: {
		react: {
			version: 'detect',
		},
	},
};
