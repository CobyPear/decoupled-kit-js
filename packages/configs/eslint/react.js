module.exports = {
	plugins: ['react'],
	extends: [
		'./index.js',
		'plugin:react/recommended',
		'plugin:react-hooks/recommended',
	],
	settings: {
		react: {
			version: 'detect',
		},
	},
};
