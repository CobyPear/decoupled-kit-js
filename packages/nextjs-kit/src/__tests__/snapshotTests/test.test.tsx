import { render } from '@testing-library/react';

import Test from '../../components/test';

/**
 * @vitest-environment jsdom
 */

describe('<Test />', () => {
  it("should render 'test'", () => {
    const { asFragment } = render(<Test greeting="hello" title="world" />);

    expect(asFragment()).toMatchSnapshot();
  });
});
