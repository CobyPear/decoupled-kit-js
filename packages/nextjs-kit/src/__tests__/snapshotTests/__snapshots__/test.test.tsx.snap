// Vitest Snapshot v1

exports[`<Test /> > should render 'test' 1`] = `
<DocumentFragment>
  <h2>
    world
  </h2>
  <h3>
    hello
  </h3>
</DocumentFragment>
`;
